functions/xnetwork/main.k

```diff
@@ -9,20 +8,14 @@ _ocds = option("params").ocds
 _dxr = option("params").dxr
 dcds = option("params").dcds
 
-providerConfigRefName = oxr.spec.parameters.providerConfigName or None
+params = oxr.spec.parameters    # extract parameter values from XR
 
 _metadata = lambda name = str -> any {
     {
         annotations = {"krm.kcl.dev/composition-resource-name" = name}
     }
 }
 
-_defaults = {
-    deletionPolicy = option("params")?.oxr?.spec.parameters.deletionPolicy or "Delete"
-    if providerConfigRefName:
-        providerConfigRef.name = providerConfigRefName
-}
```

Comment:
related to the above comment, if we expose reasonable managementPolicies as default it will be not required